Rakefile

```diff
@@ -21,14 +23,17 @@ require 'bundler_api/gem_helper'
 $stdout.sync = true
 Thread.abort_on_exception = true
 
+RuboCop::RakeTask.new
```

Comment:
this line (and the require) need to go inside the block, so a `LoadError` can be rescued